configure.ac

```diff
@@ -210,6 +210,7 @@ dnl Check for providers
 build_providers=
 
 dnl Standard providers
+ENCHANT_CHECK_PKG_CONFIG_PROVIDER([nuspell], [NUSPELL])
```

Comment:
Please use same order for spelling checkers as elsewhere.

Reply:
Fixed in https://github.com/nuspell/enchant/commit/205c4b8423d3140a1c6f3e3e77f0841145c7fb69#diff-67e997bcfdac55191033d57a16d1408a